src/services/backend.js

```diff
@@ -123,6 +109,48 @@ const sendToBetaBroker = (sessionId, chunks, genesisHash) =>
     ).then(resolve);
   });
 
+const confirmPaid = (host, id) => { //change to confirmPaid
+  return new Promise((resolve, reject) => {
+    axiosInstance
+      .get(`${host}${API.V2_UPLOAD_SESSIONS_PATH}/${id}`)
+      .then(response => {
+        resolve(response.data.paymentStatus);
+      })
+      .catch(error => {
+        reject(error);
+      });
+  });
+};
+
+const initializeUploadSession = (chunks, fileName, handle) => {
+  const host = API.BROKER_NODE_A;
+  const genesisHash = Encryption.genesisHash(handle);
+  const numChunks = chunks.length;
+  const storageLengthInYears = 999; //TODO make this a real thing
```

Comment:
Would now be a good time to get the real storageLength?

Reply:
yeah, good point. I will wire that up to the slider.